Entities/Attachable.cpp

```diff
@@ -125,7 +125,10 @@ namespace RTE {
 		} else if (propName == "InheritedRotAngleRadOffset" || propName == "InheritedRotAngleOffset") {
 			reader >> m_InheritedRotAngleOffset;
 		} else if (propName == "InheritedRotAngleDegOffset") {
+			const char* locale = std::setlocale(LC_ALL, nullptr);
+			std::setlocale(LC_ALL, "C");
```

Comment:
I wonder if we could just setlocale to "C" or "en_US.UTF-8" when initializing the game and not have to worry about it all over the place? It's not like the game supports any other languages or anything.

Reply:
doesn't work for some reason, apparently allegro occasionally resets the locale